src/js/view/controls/shortcuts-tooltip.ts

```diff
@@ -116,14 +116,18 @@ export default function (
     };
 
     const documentClickHandler = (e: Event) => {
-        const target = e.target as HTMLElement;
-        if (!/jw-shortcuts|jw-switch/.test(target.className)) {
+        const target: EventTarget | null = e.target;
+        if (target instanceof HTMLElement && !/jw-shortcuts|jw-switch/.test(target.className)) {
```

Comment:
Can we use `!== null` or simply `target` here?

Reply:
If `Event` `target` and `currentTarget` are either an element or null, it's much cheaper to just test that it's not `null` or truthy. We don't have a dependency on `HTMLElement` as global, or minor performance impact of looking up the prototype chain.

Reply:
`e.target` is an `EventTarget` when defined, which isn't necessarily an HTMLElement. While instanceof does incur about a 1/24000 seconds cost to check, this is comparable to checking if it has classname and ensures that this is a type-safe function. That being said, if we don't mind reading e.target twice there's a cleaner way to write it. See latest commit @robwalch .